### ISSUE TITLE
Fundamentals of Neural Networks

### ISSUE THREAD

**Issue by mraymer** [rballentine25/learning-python#25]
- [ ] Perceptrons, MLPs and feed-forward neural networks
- [ ] Back-propagation and network training
- [ ] Optimization via stochastic gradient descent
- [ ] Optimization meta-parameters: ADAM, etc.
